.github/workflows/pull-requests.yml

```diff
@@ -0,0 +1,49 @@
+name: PR Validation
```

Comment:
Why not use the same filename so it is easier to track what has changed?
`.github/workflows/continuous-delivery-pipeline.yml`

Reply:
Too generic, in my opinion. For example, I would put the release action in another flow. Maybe we can settle on `check` or `test`?